plugins/kubernetes/test/models/watchers/cluster_pod_watcher_test.rb

```diff
@@ -66,54 +71,65 @@
-        actor(:watcher).terminate
+        actor(watcher_name).terminate
       end
 
       it 'gets restarted after a crash' do
-        send_notice(actor(:stream), project_name)
+        send_notice(actor(:stream))
         wait_for_notice_count(actor(:subscriber), 1)
 
-        id_before = actor(:watcher).wrapped_object.object_id
+        id_before = actor(watcher_name).wrapped_object.object_id
         send_broken_notice(actor(:stream))
 
-        send_notice(actor(:stream), project_name)
+        send_notice(actor(:stream))
         wait_for_notice_count(actor(:subscriber), 2)
-        refute_equal(id_before, actor(:watcher).wrapped_object.object_id)
+        refute_equal(id_before, actor(watcher_name).wrapped_object.object_id)
       end
     end
 
     describe 'managing watchers' do
+      let(:cluster) { kubernetes_clusters(:test_cluster) }
+
       before do
         Kubernetes::Cluster.any_instance.stubs(:client).returns(DummyClient.new(Celluloid::Actor[:stream]))
       end
       after do
-        actor(:cluster_pod_watcher_1).terminate
+        actor(cluster_name(cluster)).terminate
       end
 
       it 'starts a watcher for a given cluster' do
-        Watchers::ClusterPodWatcher::start_watcher(kubernetes_clusters(:test_cluster))
-        assert actor(:cluster_pod_watcher_1).alive?
+        Watchers::ClusterPodWatcher::start_watcher(cluster)
+        assert actor(cluster_name(cluster)).alive?
       end
 
       it 'restarts a watcher for a given cluster' do
-        Watchers::ClusterPodWatcher::start_watcher(kubernetes_clusters(:test_cluster))
-        assert actor(:cluster_pod_watcher_1).alive?
-        id_before = actor(:cluster_pod_watcher_1).wrapped_object.object_id
-        Watchers::ClusterPodWatcher::restart_watcher(kubernetes_clusters(:test_cluster))
-        assert actor(:cluster_pod_watcher_1).alive?
-        refute_equal(id_before, actor(:cluster_pod_watcher_1).wrapped_object.object_id)
+        Watchers::ClusterPodWatcher.start_watcher(cluster)
+        assert actor(cluster_name(cluster)).alive?
+        id_before = actor(cluster_name(cluster)).wrapped_object.object_id
+        Watchers::ClusterPodWatcher.restart_watcher(cluster)
+        assert actor(cluster_name(cluster)).alive?
+        refute_equal(id_before, actor(cluster_name(cluster)).wrapped_object.object_id)
       end
     end
   end
 
   describe 'as a class' do
-    it 'builds a correct pod watcher name' do
-      assert_equal(:cluster_pod_watcher_1,
-                   Watchers::ClusterPodWatcher.pod_watcher_symbol(kubernetes_clusters(:test_cluster)))
+    let(:cluster) { kubernetes_clusters(:test_cluster) }
+
+    it 'builds a correct ClusterPodWatcher name' do
+      assert_equal(cluster_name(cluster).to_sym, Watchers::ClusterPodWatcher.watcher_symbol(cluster))
+    end
+
+    it 'builds a correct ClusterPodErrorWatcher watcher name' do
+      assert_equal(cluster_name(cluster).to_sym, Watchers::ClusterPodWatcher.watcher_symbol(cluster))
```

Comment:
You caught me. Damn.

Nah, you're right, I was not thinking when I did this. As you said... copy/paste... Shame on me. :)